lib/creator/yeoman/utils/entry.js

```diff
@@ -41,7 +41,7 @@ module.exports = (self, answer) => {
 			return forEachPromise(entryIdentifiers, (entryProp) => self.prompt([
 				InputValidate(
 					`${entryProp}`,
-					`What is the location of '${entryProp}'? [example: '../dist/app.js']`,
+					`What is the location of '${entryProp}'? [example: './app']`,
```

Comment:
Wasn't the entire point of this PR to change the example from `./dist` to something else? If so, I think it would be better with `./build/app`!